app/views/branches/show.json.erb

```diff
@@ -0,0 +1,5 @@
+<%
+  json_data = @branch.attributes
+  json_data['builds'] = @builds
```

Comment:
I am in favor of calling this attribute `recent_builds` as that is more accurate.